runtime/util_core/vmargs_core.c

```diff
@@ -71,7 +71,7 @@ newJavaVMArgInfo(J9JavaVMArgInfoList *vmArgumentsList, char *optString, uintptr_
  */
 
 intptr_t
-parseOptionsFileText(J9PortLibrary* portLibrary, const char* fileText, J9JavaVMArgInfoList *vmArgumentsList, uintptr_t verboseFlags)
+parseOptionsFileText(J9PortLibrary* portLibrary, const char* fileText, J9JavaVMArgInfoList *vmArgumentsList, uintptr_t verboseFlags, UDATA isDefault)
```

Comment:
Make this a more generic name, like `containsUserArgs`.

Reply:
This option is set to TRUE if it is a default option file. Perhaps a name like "containsDefaultArgs" might be less confusing.